cupyx/_runtime.py

```diff
@@ -111,8 +111,9 @@ def __init__(self, *, full=True):
         # CUDA Driver
         self.cuda_build_version = str(cupy.cuda.driver.get_build_version())
         if cupy.cuda.driver._is_cuda_python():
-            import cuda
-            self.cuda_build_version += f' (CUDA Python: {cuda.__version__})'
+            import cuda.bindings
```

Comment:
We will need a try-except here. This only works if users are using latest cuda-python 11.8.5 or 12.6.2. Older versions don't have this new module yet.